docs/getting-started.md

```diff
@@ -70,18 +72,23 @@ Open `package.json` and add the following `scripts`:
 
 These scripts refer to the different stages of developing an application:
 
-- `dev` - Runs [`next dev`](/docs/api-reference/cli.md#development) which starts Next.js in development mode
-- `build` - Runs [`next build`](/docs/api-reference/cli.md#build) which builds the application for production usage
-- `start` - Runs [`next start`](/docs/api-reference/cli.md#production) which starts a Next.js production server
-- `lint` - Runs [`next lint`](/docs/api-reference/cli.md#lint) which sets up Next.js' built-in ESLint configuration
+- `dev` - Runs [`next dev`](/docs/api-reference/cli.md#development) to start Next.js in development mode
+- `build` - Runs [`next build`](/docs/api-reference/cli.md#build) to build the application for production usage
+- `start` - Runs [`next start`](/docs/api-reference/cli.md#production) to start a Next.js production server
+- `lint` - Runs [`next lint`](/docs/api-reference/cli.md#lint) to set up Next.js' built-in ESLint configuration
+
+Create two directories `pages` and `public` at the root of your application:
 
-Next.js is built around the concept of [pages](/docs/basic-features/pages.md). A page is a [React Component](https://reactjs.org/docs/components-and-props.html) exported from a `.js`, `.jsx`, `.ts`, or `.tsx` file in the `pages` directory.
+- `pages` - Associated with a route based on their file name. For example `pages/about.js` is mapped to `/about`
+- `public` - Stores static assets such as images, fonts, etc. Files inside `public` directory can then be referenced by your code starting from the base URL (`/`).
 
-Pages are associated with a route based on their file name. For example `pages/about.js` is mapped to `/about`. You can even add dynamic route parameters with the filename.
+Next.js is built around the concept of [pages](/docs/basic-features/pages.md). A page is a [React Component](https://reactjs.org/docs/components-and-props.html) exported from a `.js`, `.jsx`, `.ts`, or `.tsx` file in the `pages` directory. You can even add [dynamic route](/docs/routing/dynamic-routes) parameters with the filename.
 
-Create a `pages` directory inside your project.
+Inside the `pages` directory add the following file to get started:
```

Comment:
```suggestion
Create a `pages` directory containing a `index.js` file to get started:
```

Lets keep "Create" since you need to create both a directory and the file.

Reply:
We are already creating `pages` and `public` directories. Please see the line `80`.